packages/client/src/store/commonSlice.ts

```diff
@@ -0,0 +1,34 @@
+import { createSlice } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+interface CommonState {
+  currentUser: string,
+  theme: string,
+  language: string,
+}
+
+const initialState: CommonState = {
+  currentUser: 'Yolo man',
+  theme: 'default',
+  language: 'RU',
+};
+
+export const counterSlice = createSlice({
+  name: 'common',
+  initialState,
+  reducers: {
+    setUser: (state, action) => {
+      state.currentUser = action.payload;
+    },
+    setTheme: (state, action) => {
```

Comment:
payload у экшенов сейчас с типом any. Надо типизировать через PayloadAction

И нужно вынести тип для темы, пользователя и языка.
```
export type TTheme = 'default' // export type TTheme = 'default' | 'dark' | 'light'
export type TUser = {
  name: string,
  id: number, 
  // ...
}
export type TLanguage = 'ru' | 'en'
```

```suggestion
    setTheme: (state, action: PayloadAction<TTheme>) => {
```

Reply:
CurrentUser наверное нужен объект, вдруг далее что то еще туда занесем. Как посчитаешь нужным)